fluent-bundle/src/bundle.rs

```diff
@@ -238,6 +240,17 @@ impl<R> FluentBundle<R> {
         self.use_isolating = value;
     }
 
+    pub fn set_transform<F>(&mut self, func: Option<F>)
+    where
+        F: 'static + Fn(&str) -> Cow<str> + Send + Sync
+    {
+        if let Some(f) = func {
```

Comment:
this whole thing can be written as `self.transform = func.map(Box::new)`

Reply:
```rust
error[E0308]: mismatched types
   --> fluent-bundle/src/bundle.rs:247:26
    |
247 |         self.transform = func.map(Box::new)
    |                          ^^^^^^^^^^^^^^^^^^ expected trait std::ops::Fn, found type parameter
    |
    = note: expected type `std::option::Option<std::boxed::Box<(dyn for<'r> std::ops::Fn(&'r str) -> std::borrow::Cow<'r, str> + std::marker::Send + std::marker::Sync + 'static)>>`
               found type `std::option::Option<std::boxed::Box<F>>`
```